### ISSUE TITLE
Hashtags

### ISSUE THREAD

**Issue by brandudno** [sonnyt/Tweetie#46]
I'm having issues with hashtags and documentation seems to be limited - can anyone help?

Currently I'm trying to pull through a specific hashtag using the following snippet -

      $('.tweet').twittie({

            hashtag: 'networktoolkit',
            dateFormat: '%b. %d, %Y',
            template: '<strong class="date">{{date}}</strong> - {{screen_name}} {{tweet}}',
            count: 10
        }, function () {
            setInterval(function() {
                var item = $('.tweet ul').find('li:first');

                item.animate( {marginLeft: '-220px', 'opacity': '0'}, 500, function() {
                    $(this).detach().appendTo('.tweet ul').removeAttr('style');
                });
            }, 5000);
        });

This just pulls through my twitter feed with no reference to the hashtag. What am i doing wrong?

**Comment by dan-lyn** [comment]
Your snippet is correct. The Twitter Search API just doesn't seem to go as far back in time as the last tweet containing "#networktoolkit". I tested it by writing a new tweet with "#networktoolkit" as a hashtag and it showed up in the feed - and yes, I used the same snippet as you above. Try using another tag or writing new tweets containing the hashtag and you will see, that it works fine.

**Comment by brandudno** [comment]
Thanks for getting back to me so promptly! 

I've had a quick look and I'm still not sure it's working 100%.

Does this work for anyone who tweets that hashtag? I only want tweets from anybody  who contain that hashtag

**Comment by dan-lyn** [comment]
As I wrote: "I tested it by writing a new tweet with "#networktoolkit" as a hashtag and it showed up in the feed". Try that yourself.

**Comment by brandudno** [comment]
Thanks, just tried this again and it just seems to be pulling through all tweets regardless of hashtag.

**Comment by dan-lyn** [comment]
Problem is somewhere else then. I'm having no problems with it. Can I see it somewhere?

**Comment by brandudno** [comment]
Yep, 

http://digitronixcloud.co.uk/dev/cihm-toolkit/

Scroll down just a little and you should see the feed. 

At the moment, I'm just trying to pull through a trending hashtag so it's current.

  $('.tweet').twittie({
   
			
            hashtag: 'PremiosGoya',
            template: '<div class="single-tweet"><p>{{tweet}}</p><br /> <span class="username">{{screen_name}}</span></div>',
            count: 10
        }, function () {
            setInterval(function() {
                var item = $('.tweet ul').find('li:first');

                item.animate( {marginLeft: '-780px', 'opacity': '0'}, 500, function() {
                    $(this).detach().appendTo('.tweet ul').removeAttr('style');
                });
            }, 5000);
        });

**Comment by dan-lyn** [comment]
I copied your script and it works.

Maybe you have a caching problem?
It seems you are requesting all tweets with: http://digitronixcloud.co.uk/dev/cihm-toolkit/wp-content/themes/nhs-network-tool/assets/api/tweet.php?username=&list=&count=10&exclude_replies=false
But your code doesn't match this request. Clear caches and try again.

**Comment by brandudno** [comment]
No luck /:

Could the server be caching?

I noticed if I add &hashtag to that url it pulls through all tweets

http://digitronixcloud.co.uk/dev/cihm-toolkit/wp-content/themes/nhs-network-tool/assets/api/tweet.php?username=&list=&count=5&exclude_replies=false&hashtag=PremiosGoya

**Comment by dan-lyn** [comment]
Yes, it could. You could also try fixing all the js errors on the page. Anyways the problem is not Tweetie.

**Comment by dan-lyn** [comment]
The same code. Just try it out locally or somewhere else instead.
